templates/vue/src/components/modals/NodeModal/forms/ContentForm/ActivityForm.vue

```diff
@@ -39,25 +39,18 @@
             class="mb-3"
           >
             <b-form-group>
-              <b-form-checkbox
-                v-model="question.followUp.enabled"
-                switch
-                @change="togglingFollowUp(question)"
-              >
-                {{ question.followUp.enabled ? "Yes" : "No" }}
+              <b-form-checkbox v-model="question.isFollowUp" switch>
+                {{ question.isFollowUp ? "Yes" : "No" }}
```

Comment:
Use of question.followUp.enabled is correct

Reply:
patched